EventFilter/RPCRawToDigi/src/RPCAMCLinkEvents.cc

```diff
@@ -0,0 +1,78 @@
+#include "EventFilter/RPCRawToDigi/interface/RPCAMCLinkEvents.h"
+
+#include "EventFilter/RPCRawToDigi/interface/RPCAMCLinkEvent.h"
```

Comment:
This file is already included in the header, no need to repeat it here

Reply:
Thank you. I'll start from cleaning up headers.